Content.Server/Speech/EntitySystems/RussianAccentSystem.cs

```diff
@@ -0,0 +1,43 @@
+using System.Text;
+using Content.Server.Speech.Components;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class RussianAccentSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<RussianAccentComponent, AccentGetEvent>(OnAccent);
+    }
+
+    public string Accentuate(string message)
+    {
+        var accentedMessage = new StringBuilder(message);
+
+        accentedMessage.Replace('b', 'в');
+        accentedMessage.Replace('N', 'И');
+        accentedMessage.Replace('n', 'и');
+        accentedMessage.Replace('K', 'К');
+        accentedMessage.Replace('k', 'к');
+        accentedMessage.Replace('m', 'м');
+        accentedMessage.Replace('h', 'н');
+        accentedMessage.Replace('t', 'т');
+        accentedMessage.Replace('R', 'Я');
+        accentedMessage.Replace('r', 'я');
+        accentedMessage.Replace('Y', 'У');
+        accentedMessage.Replace('W', 'Ш');
+        accentedMessage.Replace('w', 'ш');
```

Comment:
This should use a list of replacements, instead of of duplicated code.

Reply:
I'm not sure how Replace works internally but a switch statement might be better given the volume?